backend/dataall/base/utils/naming_convention.py

```diff
@@ -23,9 +23,10 @@ class NamingConventionPattern(Enum):
     NOTEBOOK = {'regex': '[^a-zA-Z0-9-]', 'separator': '-', 'max_length': 63}
     MLSTUDIO_DOMAIN = {'regex': '[^a-zA-Z0-9-]', 'separator': '-', 'max_length': 63}
     DEFAULT = {'regex': '[^a-zA-Z0-9-_]', 'separator': '-', 'max_length': 63}
+    DEFAULT_SEARCH = {'regex': '[^a-zA-Z0-9-_ ]', 'separator': '-', 'max_length': 1000}
     OPENSEARCH = {'regex': '[^a-z0-9-]', 'separator': '-', 'max_length': 27}
     OPENSEARCH_SERVERLESS = {'regex': '[^a-z0-9-]', 'separator': '-', 'max_length': 31}
-    DATA_FILTERS = {'regex': '^[a-z0-9_]*$', 'separator': '_', 'max_length': 31}
+    DATA_FILTERS = {'regex': '[^a-z0-9_]', 'separator': '_', 'max_length': 31}
```

Comment:
note: `DATA_FILTERS` was formerly only using the `validate_name` pattern and was the the only `pattern` with a `regex` value that was the desired character pattern rather than the not allowed set of chars

- (1) Changed `DATA_FILTERS` Pattern to have `regex` representing not allowed characters
- (2) Changed re.search condition in `validate_name` to match